lib/resources/r4/person.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: Person
+field_name_base_url: https://hl7.org/fhir/R4/person-definitions.html#Person
+fields:
+- name: identifier
+  type: List of Identifier
+  description: The identifier for the person.
+  action: terminology
+  children:
+
+  - name: type
+    type: CodeableConcept
+    description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+    url: http://hl7.org/fhir/r4/datatypes-definitions.html#Identifier.type
```

Comment:
Secure link here as well. Use 'https'

Reply:
Changed in commit 8f913dce83505df0c6e3815d58fb4cf8367d8b44